### PR TITLE
add jakartams to sbt project list

### PR DESCRIPTION
**Pull request by pjfanning**
Added in #674 but this sbt issue seems to stop the jar being published

see https://repository.apache.org/content/groups/snapshots/org/apache/pekko/ - no jakartams there

* also tidies up imports
* fixes doc build issues - the class names in the new connector clash with those in the jms connector 
* fix mima issue - there is no previous jar to compare against

### CODE REVIEW

**Comment by pjfanning** [comment]
@samueleresca the new #674 classes are breaking the doc build. See the checks for this PR.

It looks hard to fix. Paradox has built-in support for having 2 classes of the same name but one is in javadsl and one is in scaladsl. This support can't handle having a 2nd set of classes with the same name.

Would it be possible to rename all the `Jms` prefixed classes in the new connector to have something like `JakartaMs` or `JakartaMessaging` or `JakartaJms` prefix?

**Comment by pjfanning** [comment]
@samueleresca I think I have found a way to fix the doc issue - just testing it locally right now

File: jakartams/src/main/scala/org/apache/pekko/stream/connectors/jakartams/impl/JmsInternalMatValues.scala [hunk 1]
```diff
@@ -14,10 +14,10 @@
...
 
 import org.apache.pekko
-import org.apache.pekko.NotUsed
-import org.apache.pekko.annotation.InternalApi
-import org.apache.pekko.stream.KillSwitch
-import org.apache.pekko.stream.scaladsl.Source
+import pekko.NotUsed
+import pekko.annotation.InternalApi
+import pekko.stream.KillSwitch
+import pekko.stream.scaladsl.Source
```
Comment (He-Pin) [inline jakartams/src/main/scala/org/apache/pekko/stream/connectors/jakartams/impl/JmsInternalMatValues.scala:20]: Maybe better if sort the imports

**Review by He-Pin — APPROVED** [review]
Lgtm

File: jakartams/src/main/scala/org/apache/pekko/stream/connectors/jakartams/impl/JmsInternalMatValues.scala (hunk 1 above)
Comment (pjfanning) [reply jakartams/src/main/scala/org/apache/pekko/stream/connectors/jakartams/impl/JmsInternalMatValues.scala:20]: these look sorted to me already

**Comment by samueleresca** [comment]
> @samueleresca the new #674 classes are breaking the doc build. See the checks for this PR.
> 
> It looks hard to fix. Paradox has built-in support for having 2 classes of the same name but one is in javadsl and one is in scaladsl. This support can't handle having a 2nd set of classes with the same name. The issue is if we try to use the FQCN, we will need to include the full package name and this negates the Paradox support for linking the javadsl/scaladsl version of the class depending on whether the UI user has chosen Java or Scala.
> 
> Would it be possible to rename all the `Jms` prefixed classes in the new connector to have something like `JakartaMs` or `JakartaMessaging` or `JakartaJms` prefix?

Why the build wasn't catching this at PR level? (I haven't seen anything failing). Let me come back on this in the next few days. I can rename the classes and give it a try again. Sorry for the inconvenience

**Comment by pjfanning** [comment]
@samueleresca I think this PR fixes the issues. The main build issue was the fact this new module wasn't added to the project list in build.sbt. This means that non-scoped sbt comands will ignore the module. So `sbt jakartams/test` works but `sbt publishM2` ignored jakartams.

**Pull request merged**